docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Extend_CMS_Interface.md

```diff
@@ -642,6 +642,146 @@ class MyGallery extends React.Component {
 export default withInjector(MyGallery);
 ```
 
+### Using injector to customise redux state data
+
+It is recommended to be familiar with the concepts of Immutability and Redux before starting this.
+
+The examples use `Spread in object literals` which is at this moment in Stage 3 Proposal. If you're more comfortable with using
```

Comment:
Link to description of this syntax? It's a bit hard to tell what you mean without a code example. Either point out that it's `{...}` syntax, or only mention this after first using the syntax in an example. At the moment, it'll feel to devs like they're missing something important that's hidden in the examples.